src/router/index.ts

```diff
@@ -12,7 +12,8 @@ import CreateUpdateThresholdRule from '@/views/CreateUpdateThresholdRule.vue';
 import CreateUpdateSafetyStockRule from '@/views/CreateUpdateSafetyStockRule.vue'
 import CreateUpdateStorePickupRule from '@/views/CreateUpdateStorePickupRule.vue'
 import CreateUpdateShippingRule from '@/views/CreateUpdateShippingRule.vue'
-
+import { DxpLogin, useAuthStore } from '@hotwax/dxp-components';
```

Comment:
Remove the import for app's login page and also remove Login page from the app, as we are using DxpLogin.